v2/blockstore/readwrite_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"testing"
 	"time"
 
+	dag "github.com/ipfs/go-merkledag"
```

Comment:
I think merkledag would be better FYI, for clarity

Reply:
`dag` was used consistently across tests; replaced all with `merkledag`.